postman-generator/src/test/scala/generator/DependantOperationResolverSpec.scala

```diff
@@ -17,13 +19,11 @@ class DependantOperationResolverSpec extends WordSpec with Matchers {
       val resolvedService = ServiceImportResolver.resolveService(testMainService, Seq(referenceApiService))
       val result = DependantOperationResolver.resolve(resolvedService)
 
-      result shouldEqual Seq(
-        objectRef1AttrValue -> dependency1Target
-      )
+      result.head should === (objectRef1AttrValue.toExtended -> dependency1Target)
```

Comment:
the previous version wouldn't throw `NoSuchElementException` because of `None.get` :)